core/trino-main/src/test/java/io/trino/sql/planner/planprinter/TestJsonRepresentation.java

```diff
@@ -84,7 +84,7 @@ public void testLogicalJsonPlan()
                 ImmutableList.of(new JsonRenderedNode(
                         "98",
                         "Limit",
-                        ImmutableMap.of("count", "10", "withTies", "", "inputPreSortedBy", "[]"),
```

Comment:
nit: Why this has changed? should we preserve `[]` notation for explicitly marking empty list?